TOMP-API.yaml

```diff
@@ -1160,49 +1161,46 @@ components:
         legs:
           type: array
           items:
-            $ref: '#/components/schemas/operator-leg'
+            $ref: '#/components/schemas/operatorLeg'
         conditions:
           description: references to the 'conditions' array (start of this object).
           type: array
           items:
             type: string
 
-    operator-leg:
+    operatorLeg:
       type: object
-      allOf: 
-        - $ref: '#/components/schemas/simple-leg'
+      allOf:
+        - $ref: '#/components/schemas/simpleLeg'
       properties:
-        operator-name:
+        operatorName:
           type: string
-        operator-maas-id: 
+        operatorMaasId:
           type: string
-          description: the maas-id from the operator
-        operator-description:
+          description: the maasId from the operator
+        operatorDescription:
           type: string
           description: short description of the operator
-        operator-contact:
+        operatorContact:
           type: string
           description: contact information
- 
+
     condition:
       type: object
       oneOf:
-        - $ref: '#/components/schemas/postponed-commit-condition'
-        - $ref: '#/components/schemas/return-area-condition'
-        - $ref: '#/components/schemas/deposit-condition'
-        - $ref: '#/components/schemas/upfront-payment-condition'
-        - $ref: '#/components/schemas/pay-when-finished-condition'
-        - $ref: '#/components/schemas/require-booking-data-condition'
+        - $ref: '#/components/schemas/postponedCommitCondition'
+        - $ref: '#/components/schemas/returnAreaCondition'
+        - $ref: '#/components/schemas/depositCondition'
+        - $ref: '#/components/schemas/upfrontPaymentCondition'
+        - $ref: '#/components/schemas/payWhenFinishedCondition'
+        - $ref: '#/components/schemas/requireBookingDataCondition'
       discriminator:
         propertyName: conditionType
```

Comment:
This property is not defined.

Reply:
correct. this property should be added to all the subtypes.